app/src/main/res/layout/password_fragment.xml

```diff
@@ -41,18 +42,20 @@
         app:layout_constraintEnd_toEndOf="parent"
```

Comment:
@raghav32 use `margin_start` instead of `margin_left` & `margin_end` instead of `margin_right`